editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.test.tsx

```diff
@@ -332,18 +332,108 @@ describe("basic interactions - happy path", () => {
 });
 
 describe("copy feature select", () => {
-  it.todo("is disabled if only a single feature is present");
-  // no copy select if only one feature
-  it.todo("is enabled once multiple features are present");
-  // copy select enabled once you add more features
-  it.todo(
-    "lists all other features as options (the current feature is not listed)"
-  );
-  // current tree is not an option in the copy select
-  it.todo("copies all data from one feature to another");
-  // all data fields are populated from one field to another
-  it.todo("should not have any accessibility violations");
-  // axe checks
+  it("is disabled if only a single feature is present", async () => {
+    const { getByTestId, getByTitle } = setup(<MapAndLabel {...props} />);
+    const map = getByTestId("map-and-label-map");
+
+    addFeaturesToMap(map, [point1]);
+
+    const copyTitle = getByTitle("Copy from");
+
+    const copyInput = within(copyTitle).getByRole("combobox");
+
+    expect(copyInput).toHaveAttribute("aria-disabled", "true");
+  });
+
+  it("is enabled once multiple features are present", async () => {
+    const { getByTitle } = setup(<MapAndLabel {...props} />);
+
+    addMultipleFeatures([point1, point2]);
+
+    const copyTitle = getByTitle("Copy from");
+
+    const copyInput = within(copyTitle).getByRole("combobox");
+
+    expect(copyInput).not.toHaveAttribute("aria-disabled", "true");
```

Comment:
This was the best option I could find to prove if something is disabled or not. Happy to know if there's a better option

Reply:
This is the best way - `toBeDisabled()` won't work with MUI as it's using a combobox + input instead of a native select 👍